onshape:
    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

    # Position Controllers ---------------------------------------
    wrist_pitch_position_controller:
      type: effort_controllers/JointPositionController
      joint: wrist_pitch
      pid: {p: 1.0, i: 1.0, d: 0.0}
    wrist_yaw_position_controller:
      type: effort_controllers/JointPositionController
      joint: wrist_yaw
      pid: {p: 1.0, i: 1.0, d: 0.0}
    elbow_position_controller:
      type: effort_controllers/JointPositionController
      joint: elbow
      pid: {p: 1.0, i: 1.0, d: 0.0}
    shoulder_position_controller:
       type: effort_controllers/JointPositionController
       joint: shoulder
       pid: {p: 1.0, i: 1.0, d: 0.0}
    waist_position_controller:
       type: effort_controllers/JointPositionController
       joint: waist
       pid: {p: 1.0, i: 1.0, d: 0.0}
    # To add more just add them here as the first one